cmd/snap/cmd_run.go

```diff
@@ -91,6 +92,33 @@ and environment.
 		}, nil)
 }
 
+func maybeWaitForSecurityProfileRegeneration() error {
+	// check if the security profiles key has changed, if so, we need
+	// to wait for snapd to re-generate all profiles
+	//
+	// (wait up to 1500 * 200ms = 300s)
+	timeout := 1500
+	if timeoutEnv := os.Getenv("SNAPD_DEBUG_SYSTEM_KEY_WAIT_TIMEOUT"); timeoutEnv != "" {
```

Comment:
I understand is only for tests but is a bit strange that this envvar is not in s or ms but  in # 200ms